e2e/wdio/platform/tests/page-footer.e2e-spec.ts

```diff
@@ -0,0 +1,45 @@
+import { PageFooterPo } from '../pages/page-footer.po';
+import {
+    getElementArrayLength,
+    getText,
+    isElementClickable,
+    scrollIntoView,
+} from '../../driver/wdio';
+
+describe('PageFooter test suite:', function() {
+    const pagefooterPage = new PageFooterPo();
+    const {
+        pagefooterclicablelink, pagefootertext  } = pagefooterPage;
+
+    beforeAll(() => {
+        pagefooterPage.open();
+    }, 1);
+    
+        it('verify clickable links ', () => {
+            const linksLength = getElementArrayLength(pagefooterclicablelink);
+            for (let i = 0; i < linksLength; i++) {
+                scrollIntoView(pagefooterclicablelink, i);
+                expect(isElementClickable(pagefooterclicablelink, i)).toBe(true, 'link with index ${i} not clickable');
+            }
+        });
+
+        it('should check pagefooter text', () => {
+            expect(getText(pagefootertext,)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index');
+            expect(getText(pagefootertext, 1)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index1');
+            expect(getText(pagefootertext, 2)).toEqual('1992-2021 @copyright all right reserved.', 'Text is not matching Index2');
```

Comment:
when the assertion is clear (expect to equal 'this specific text') then you don't need to add a failure message. We only do that if the assertion is true/false (expect to be true) to add clarity as to what should be happening.